### DISCUSSION TITLE
Entering in notes from the browser

### DISCUSSION THREAD

**Discussion by Sintrastes** [coddingtonbear/obsidian-web#225] [Q&A]
Hey all, I'm curious as I was not able to find this in the documentation: Is there any way I can add additional fields to a note template (for instance, if I wanted to add notes to a template from the browser extension itself)?

What I'm envisioning is that the user can specify various custom template parameters in a template, and these parameters will show in the browser extension popup as additional text fields that a user can enter in data to when they are saving a note with a particular template.

**Comment by fellnerse** [comment]
I was looking for exactly something like this as well! That was the reason for testing this plugin in the first place..
I would love to be able to have my own custom informations on browser tabs (kind of a sync between obsidian and browser)
